assets/css/global-custom.css

```diff
@@ -562,6 +567,7 @@ body{
 
 .project-card {
     border-radius: 1.5rem;
+    padding: 0px;
 }
```

Comment:
This has affected some other components to change as well. There's a slight difference in the homepage project cards now.
Shall we keep this the way it is and make a new class for this component??

Existing:
<img width="541" alt="image" src="https://user-images.githubusercontent.com/45477334/118085168-8ecfe400-b3df-11eb-892d-459df138ae57.png">
Your version:
<img width="501" alt="image" src="https://user-images.githubusercontent.com/45477334/118085420-03a31e00-b3e0-11eb-9eda-b0a9afa81118.png">

Reply:
I will create a separate class.